charts/prometheus-adapter/values.yaml

```diff
@@ -143,3 +143,10 @@ hostNetwork:
   # API server unable to communicate with metrics-server. As an example, this is required
   # if you use Weave network on EKS
   enabled: false
+
+podDisruptionBudget:
+  # Specifies if PodDisruptionBudget should be enabled
+  # When enabled, minAvailable or maxUnavailable should also be defined.
+  enabled: false
+  minAvailable:
```

Comment:
Would be good to specify defaults for these both for recommendations and to avoid nils.

Reply:
Added default for `maxUnavailable` and have condition for `minAvailable` So there will be no nils.

```yaml
# Source: prometheus-adapter/templates/custom-metrics-apiserver-pdb.yaml
apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  name: RELEASE-NAME-prometheus-adapter
  labels:
    app: prometheus-adapter
    chart: prometheus-adapter-2.7.0
    release: RELEASE-NAME
    heritage: Helm
spec:
  maxUnavailable: 1
  selector:
    matchLabels:
      app: prometheus-adapter
      release: RELEASE-NAME
```